tests/integration/it_locations_test.py

```diff
@@ -0,0 +1,62 @@
+
+from timeless.restaurants.models import Location
+from timeless.companies.models import Company
+
+
+def create_location(db_session):
+    company = Company(
+        name="Krusty Inc.",
+        code="KI",
+        address="Springfield Lane,12"
+    )
+    location =  Location(
+        name="Krusty Burger",
+        code="KB",
+        company_id=company.id,
+        country="United States",
+        region="Middle East",
+        city="Springfield",
+        address="Jebediah Street, NN",
+        longitude=23,
+        latitude=25,
+        type="B",
+        status="open",
+        comment="Fast food restaurant from a famous animated sitcom."
+    )
+    db_session.add(company)
+    db_session.add(location)
+    db_session.commit()
+    return location
+
+
+def test_create_location(db_session):
+    """Integration test for adding and selecting Location"""
+    location = create_location(db_session)
+    row = db_session.query(Location).get(location.id)
+    assert row.name == "Krusty Burger"
+
```

Comment:
didnt notice the first time. but according to pep8 there should be 2 blank lines before functions